### PR TITLE
Support PEM strings that do not end with a new line

### PR DESCRIPTION
**Pull request by blag**
Merge in @mattijohn's [changes](https://github.com/mattijohn/pem/commit/b7dbf157c61bf2331c3f93e1ab65a22340611f57).

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+certifi
```
Comment (hynek) [inline dev-requirements.txt]: this is an accident, is it?

Comment (blag) [inline dev-requirements.txt]: Yep. Fixed.

**Comment by louisnayegon** [comment]
Hi, when do you expect to merge and release this PR?

**Comment by hynek** [comment]
Sorry I missed the fix; can't release right now since I'm traveling though.

**Pull request merged**